### PR TITLE
originInfo_date_created_tesim gets better implementation

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

Almost completes #575, but there is one spec still with xit as I need more clarification from Arcadia.  This can be merged and I will keep ticket #575 open

## How was this change tested?

lots and lots of unit tests.

### CODE REVIEW

File: app/indexers/descriptive_metadata_indexer.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def to_solr
       'sw_subject_temporal_ssim' => subject_temporal,
       'sw_subject_geographic_ssim' => subject_geographic,
       'sw_pub_date_facet_ssi' => pub_year,
-      'originInfo_date_created_tesim' => [creation_date].compact,
+      'originInfo_date_created_tesim' => creation_date,
```
Comment (ndushay) [inline app/indexers/descriptive_metadata_indexer.rb:23]: Arcadia told me she preferred a single value.  I know the field can take multiple values.

File: spec/indexers/descriptive_metadata/event_date_creation_spec.rb [hunk 2]
```diff
@@ -23,9 +24,7 @@
           "hasAdminPolicy": "druid:zx485kb6348",
           "partOfProject": "H2"
         },
-        "description": {
-          #{description}
-        },
+        "description": #{JSON.generate(description)},
```
Comment (ndushay) [inline spec/indexers/descriptive_metadata/event_date_creation_spec.rb:27]: here is the magic for using ruby hashes for the descriptive metadata

File: spec/indexers/descriptive_metadata/event_date_creation_spec.rb [hunk 3]
```diff
@@ -73,363 +72,349 @@
...
+              date: [
+                {
+                  value: '1900',
+                  type: 'creation',
+                  status: 'primary'
+                }
+              ]
+            }
+          ]
+        }
```
Comment (ndushay) [inline spec/indexers/descriptive_metadata/event_date_creation_spec.rb:95]: yep - a lot of churn with the JSON -> ruby hash stuff

File: spec/indexers/descriptive_metadata/event_date_creation_spec.rb [hunk 4]
```diff
@@ -73,363 +72,349 @@
...
+      end
 
-        xit 'populates origin_info_date_created_tesim' do
-          expect(doc).to include('origin_info_date_created_tesim' => '1900')
-        end
+      it 'uses date' do
```
Comment (ndushay) [inline spec/indexers/descriptive_metadata/event_date_creation_spec.rb:98]: I switched to using a more descriptive statement in the spec description, though this is the worst example.

**Comment by ndushay** [comment]
I can only imagine the coverage drop might be from some non exercised code in event_date_builder or event_selector ... but ... that seems unlikely.  eh:  not even 1/2 a percent.

**Review by mjgiarlo — APPROVED** [review]

**Comment by ndushay** [comment]
code coverage dropped due to parallelValue code additions;  don't have specs for these.

**Comment by ndushay** [comment]
coverage drop disappeared with Arcadia tweaking xit spec and me un-xit-ing it.

**Pull request merged**